lib/testing/testrunner.js

```diff
@@ -27,6 +27,7 @@ function TestRunner(options) {
   this.provider = options.provider;
 
   this.can_shapshot = false;
+  this.first_snapshot = false;
```

Comment:
Should this be `true` to start? It seems like the `if` statement on line 81 will never execute otherwise